settings.py

```diff
@@ -44,7 +44,7 @@
 SESSION_COOKIE_DOMAIN = None
 
 # Disables custom routing in settings.py so that tasks actually run.
```

Comment:
I removed the comment but left `CELERY_ROUTES = {}` I think that just means everything runs in the same worker pool which is probably fine for development.